packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -417,14 +419,17 @@ test.describe('modal', () => {
                 expect(isModalVisible).toBe(true);
             });
 
-            test('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
+            test.skip('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
```

Comment:
Can you add a link to the ticket for this please?

Reply:
Can't link to Jira tickets in code but will drop the ticket number as a comment above.

Reply:
Given we link to JIRA tickets in PR's, I think referencing the ticket number is okay here.

Reply:
Comment added. 👍🏼